app/models/asset.rb

```diff
@@ -1,3 +1,4 @@
 class Asset < ActiveRecord::Base
   belongs_to :employee
+  has_one :categories
```

Comment:
When making it singular it breaks the program, saying that it has an association issue.